### PR TITLE
DOC-3514: Mobile doc

### PR DESCRIPTION
**Pull request by catong**
Update mobile section in learner doc and the timed exam/proctored exam sections in course author doc to specifically mention that timed and proctored exams cannot be completed using the mobile app.
Reviewer: @srpearce 
FYI: @jaakana @BenjiLee

### CODE REVIEW

File: en_us/shared/course_features/timed_exams.rst [hunk 1]
```diff
@@ -18,29 +18,29 @@ You can configure a :ref:`subsection <Developing Course Subsections>`
 in your course so that learners have a set amount of time to complete and
 submit all problems in that subsection.
 
+Although you can configure an ungraded subsection to be timed, typically you set a time limit on graded subsections, such as for mid-term or final exams.
```
Comment (srpearce) [inline en_us/shared/course_features/timed_exams.rst]: Line wrap

File: en_us/shared/students/SFD_mobile.rst [hunk 2]
```diff
@@ -188,11 +188,21 @@ Course Questions
...
+The answer depends on the types of problems that are used in the course you
+are taking. If a course contains mobile friendly problem types, you can
+complete those types of problems using the mobile app. For a list of the types
+of problems you can complete using the mobile app, see :ref:`What problems in
+mobile app`. Some exams, specifically timed or proctored exams, cannot be
+taken using the mobile app.
+
+You can watch course videos on the mobile app, and can also download videos to
+your mobile device for watching when you do not have an internet connection.
+When you have an Internet connection, you can also read course announcements,
```
Comment (srpearce) [inline en_us/shared/students/SFD_mobile.rst]: Lowercase Internet

File: en_us/shared/students/SFD_mobile.rst [hunk 3]
```diff
@@ -188,11 +188,21 @@ Course Questions
...
+participate in course discussions, and complete the mobile friendly problems
+in your assignments.
+
+To complete an entire course, it is best to use a web browser on a computer.
```
Comment (srpearce) [inline en_us/shared/students/SFD_mobile.rst]: I don't know that I would say this—it kind of sounds like we're discouraging people from using the mobile app. Perhaps "You can complete all course-related actions in a web browser on a computer"?

File: en_us/shared/students/SFD_mobile.rst [hunk 4]
```diff
@@ -188,11 +188,21 @@ Course Questions
...
-Not entirely. You can download course videos to watch in the mobile app when you
-do not have an Internet connection. When you have an Internet connection, you
-can also read course announcements and content, participate in course
-discussions, and do some, but not all, of the problems in your assignments. To
-complete an entire course, you use a web browser on a computer.
+The answer depends on the types of problems that are used in the course you
+are taking. If a course contains mobile friendly problem types, you can
+complete those types of problems using the mobile app. For a list of the types
+of problems you can complete using the mobile app, see :ref:`What problems in
+mobile app`. Some exams, specifically timed or proctored exams, cannot be
```
Comment (srpearce) [inline en_us/shared/students/SFD_mobile.rst]: Nit: I feel like there should be a comma after specifically, but that's a little hard to do here...

Some exams (specifically, timed or proctored exams) cannot be

Or change "specifically" to "such as"

**Review by srpearce — APPROVED** [review]
A couple of minor things.

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1370

**Pull request merged**